packages/components/src/components/date-picker/date-picker.scss

```diff
@@ -38,6 +38,9 @@
             text-align: center;
         }
     }
+    &__icon {
```

Comment:
Perhaps we can combine both `rules` in `dc-icon` and `dc-datepicker__icon`?

Reply:
Yes, thanks for notifying :)